.github/workflows/wp-scripts-lint.yml

```
@@ -135,7 +132,7 @@ jobs:
 
       - name: Lint style files
         if: ${{ contains(fromJSON(inputs.LINT_TOOLS), 'style') }}
-        run: ./node_modules/.bin/wp-scripts lint-style ${{ inputs.STYLELINT_ARGS }}
+        run: ./node_modules/.bin/wp-scripts lint-style ${{ inputs.STYLELINT_ARGS }} --formatter github
```

In theory, this should work.

```suggestion
        run: echo ${INPUT_STYLELINT_ARGS//'--formatter github'/} | xargs ./node_modules/.bin/wp-scripts lint-style --formatter github
```
 The single problem could be https://github.com/orgs/community/discussions/27088. So the direct destruction might be required with:
```yaml
env:
   INPUT_STYLELINT_ARGS: ${{ inputs.STYLELINT_ARGS }}
```

---

We discussed this internally and agreed not to workaround cases where `--formatter` is passed in `INPUT_STYLELINT_ARGS` because only very few packages are affected: https://github.com/search?q=org%3Ainpsyde+OR+org%3AInpsyde-Global-Service-Provider+STYLELINT_ARGS+--formatter&type=code.
These will be updated once they are worked on next.